tests/fileTests.py

```diff
@@ -18,6 +18,7 @@ class FileTests(BaseTestAPI):
     updateSubmissionId = None
     filesSubmitted = False
     submitFilesResponse = None
+    status_check_time = {"year":2016,"month":01,"day":02,"hour":03,"minute":04,"second":05}
```

Comment:
Good catch, this was part of my attempt to manually set the last updated time, which didn't work out.  I've removed it now.